auth_backends/urls.py

```diff
@@ -2,15 +2,16 @@
 
 Add these to your project's `urlpatterns` to avoid duplicating code.
 """
-from django.conf.urls import url, include
+from django.conf.urls import include
+from django.urls import re_path
 
 from auth_backends.views import (
     EdxOAuth2LoginView,
     EdxOAuth2LogoutView,
 )
 
 oauth2_urlpatterns = [
-    url(r'^login/$', EdxOAuth2LoginView.as_view(), name='login'),
-    url(r'^logout/$', EdxOAuth2LogoutView.as_view(), name='logout'),
-    url('', include('social_django.urls', namespace='social')),
+    re_path(r'^login/$', EdxOAuth2LoginView.as_view(), name='login'),
+    re_path(r'^logout/$', EdxOAuth2LogoutView.as_view(), name='logout'),
+    re_path('', include('social_django.urls', namespace='social')),
```

Comment:
We'll probably want to change these to use `path()` later, but I'm fine with punting that until we have more experience and docs for doing it.